src/app/pages/poll/poll.component.html

```diff
@@ -78,62 +80,60 @@
               </nz-page-header-extra>
             </nz-page-header>
             <!-- Poll item list -->
-            <ng-scrollbar class="scrollbar">
-              <div class="poll-item-list-container no-select" *ngIf="poll.pollItems.length > 0">
-                <nz-collapse class="poll-item-list" nzBordered="false" nzAccordion
-                             *ngIf="poll.pollItems !== undefined && poll.pollItems !== null && poll.pollItems.length > 0"
-                             cdkDropList (cdkDropListDropped)="drop($event)">
-                  <nz-collapse-panel class="poll-item" #p *ngFor="let pollItem of poll.pollItems" [nzHeader]="header"
-                                     [nzExtra]="options" [nzExpandedIcon]="expandedIcon" cdkDrag cdkDragLockAxis="y">
-                    <!-- Item content -->
-                    <div nz-row>
-                      <div nz-col nzSpan="9" class="stats-column">
-                        <nz-statistic [nzValue]="(getAnswersCount(pollItem.id) | number)!" nzTitle="Total answers">
-                        </nz-statistic>
-                      </div>
-                      <div nz-col nzSpan="15">
-                        <ngx-charts-bar-horizontal class="chart" [view]="[0, 300]" [results]="getChartData(pollItem.id)"
-                                                   scheme="forest" xAxis="true" yAxis="true" legend="true"
-                                                   showXAxisLabel="true" showYAxisLabel="true" animations="true"
-                                                   xAxisLabel="Participants" yAxisLabel="Answers">
-                        </ngx-charts-bar-horizontal>
-                      </div>
+            <div class="poll-item-list-container no-select" *ngIf="poll.pollItems.length > 0">
```

Comment:
could pollItems be undefined?
maybe something like: `*ngIf="poll?.pollItems?.length"`

Reply:
Good thought. I'll try this